### DISCUSSION TITLE
rtl admin panel

### DISCUSSION THREAD

**Discussion by basmalawaleed88** [filamentphp/filament#9830] [Help] [answered]
i want to allow me to change admin panel to Rtl  direction.

**Comment by leandrocfe** [comment, marked as answer]
Publish translations
```bash
php artisan vendor:publish --tag=filament-panels-translations
```
Then, configure this `'direction' => 'rtl'` in the `lang/vendor/filament-panels/[your-language]/layout.php` file

**Reply by farhadrahmanii** [reply]
Thank you so much dear leandrocfe That was fantastic help I was search for over month to find this kind of usefel solution again thank you.

**Reply by leandrocfe** [reply]
No problem ;)

**Reply by MasterAboLayan** [reply]
> Publish translations
> 
> ```shell
> php artisan vendor:publish --tag=filament-panels-translations
> ```
> 
> Then, configure this `'direction' => 'rtl'` in the `lang/vendor/filament-panels/[your-language]/layout.php` file

hi @leandrocfe 
actually, I tried to apply this but it didn't wwork form me.
could you please help me

I'm using Laraveel 11 and Filament 3.2.140

thanks

**Reply by abdallahalsamman** [reply]
@MasterAboLayan Same.

**Reply by its4nas** [reply]
@MasterAboLayan @abdallahalsamman 
you have to change the **APP_LOCALE** from **en** to **ar**

**Comment by alachiio** [comment]
it doesn't work on filament v4

**Reply by its4nas** [reply]
it works for me.

**Reply by corptia** [reply]
in AppServiceProvider.php add this in boot function
  
    public function boot(): void
    {

        FilamentView::registerRenderHook(
            'panels::body.start',
            fn (): string => '<script>document.documentElement.dir = "rtl";</script>'
        );

    }
